momento-sdk/src/main/java/momento/sdk/SendSubscribeOptions.java

```diff
@@ -10,6 +11,8 @@ class SendSubscribeOptions implements ISubscriptionCallbacks {
   ItemCallback onItem;
   CompletedCallback onCompleted;
   ErrorCallback onError;
+  DiscontinuityCallback onDiscontinuity;
+  HeartbeatCallback onHeartbeat;
```

Comment:
Any reason why these are package private vs class private?

Reply:
That might have been an oversight when we implemented topics in java sdk. 
I don't see other classes accessing these callback functions.

Reply:
We can maybe address this in a separate PR.